subiquity/controllers/filesystem.py

```diff
@@ -480,6 +483,8 @@ def delete_raid(self, raid):
         self.clear(raid)
         for p in list(raid.partitions()):
             self.delete_partition(p)
+        for d in raid.devices | raid.spare_devices:
+            d.wipe = 'superblock'
```

Comment:
This isn't strictly required; curtin already wipes both raid members and spares before using them; but I think it's prudent since the wipe setting will ensure curtin looks to see if they have any holders that need removal.

Reply:
Yeah this change was from before I read the bug properly :) It still seems conceptually valid I guess?